scipy/stats/_qmc.py

```diff
@@ -1319,47 +1323,56 @@ class Sobol(QMCEngine):
     """
 
     MAXDIM: ClassVar[int] = _MAXDIM
-    MAXBIT: ClassVar[int] = _MAXBIT
 
     def __init__(
-            self, d: IntNumber, *, scramble: bool = True,
-            seed: SeedType = None
+        self, d: IntNumber, *, scramble: bool = True,
+        bits: IntNumber = 30, seed: SeedType = None
     ) -> None:
         super().__init__(d=d, seed=seed)
         if d > self.MAXDIM:
             raise ValueError(
-                "Maximum supported dimensionality is {}.".format(self.MAXDIM)
+                f"Maximum supported dimensionality is {self.MAXDIM}."
             )
 
+        self.MAXBIT = bits
+        if self.MAXBIT > 64:
```

Comment:
`MAXBIT` sounds like it is always going to be 64. Why is this not just `self.bits`, and the argument to `_initialize_v` not just `bits`?